.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea/
```

Comment:
This should not be in here - if you want to ignore IDE files, put them into your global gitignore file